src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Wysiwyg.Edit.cshtml

```diff
@@ -18,12 +18,12 @@
     @if (culture.IsRightToLeft())
     {
         <div style="text-align:right">
-            <textarea asp-for="Html" class="form-control"></textarea>
```

Comment:
Why the bootstrap class removed, is it because Trumbowyg will override the textarea?

Reply:
Yes